src-ghc/Pact/Interpreter.hs

```diff
@@ -185,7 +185,7 @@ setupEvalEnv
   -> ExecutionConfig
   -> IO (EvalEnv e)
 setupEvalEnv dbEnv ent mode msgData refStore gasEnv np spv pd ec = do
-  gasRef <- newIORef 0
+  gasRef <- newIORef (MilliGas 0)
```

Comment:
```suggestion
  gasRef <- newIORef mempty
```

Reply:
Ah I made it explicit on purpose, but I'll revert it sure.